manifests/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -714,6 +714,17 @@ data:
             {{ if (isset .ObjectMeta.Annotations `sidecar.istio.io/proxyMemory`) -}}
             memory: "{{ index .ObjectMeta.Annotations `sidecar.istio.io/proxyMemory` }}"
             {{ end }}
+          limits:
```

Comment:
The logic here doesn't seem quite right, as this block will not be hit unless `sidecar.istio.io/proxyCPU` is set. You may not want to set that annotation?

More so the whole logic is actually kind of broken, since we only support annotations OR defaults it looks like, when ideally the annotations just override the defaults for all 4 fields

Reply:
The logic is indeed kind of weird, but I tried to improve it in such a way that at least the limits are always set. Indeed, with this logic you cannot just set the limits and not the requests.
What do you suggest to improve here?

Reply:
I guess thinking more about it, you probably don't want to set limits without requests. If you do, you can just set the request to the same as the limit and the result is the same. I was thinking that you could set the limit and keep the default request, but that doesn't actually happen, so I think this is probably fine

Reply:
Ok, if there's anything else I need to do, let me know.
Would it be possible to get this into a next 1.4/1.5 release as well?